lib/full_text_search/custom_field_mapper.rb

```diff
@@ -0,0 +1,36 @@
+module FullTextSearch
+  class CustomFieldMapper < Mapper
+    class << self
+      def redmine_mapper_class
+        RedmineCustomFieldMapper
+      end
+
+      def fts_mapper_class
+        FtsCustomFieldMapper
+      end
+    end
+  end
+  resolver.register(CustomField, CustomFieldMapper)
+
+  class RedmineCustomFieldMapper < RedmineMapper
+    class << self
+      def with_project(redmine_class)
```

Comment:
Question: Do we something in `with_project`? I think this Mapper is not be used with project because custom field is independent from project.

Reply:
`with_project` is for mapper that needs `JOIN` to collect related projects.